addon/templates/components/gavant-table.hbs

```diff
@@ -0,0 +1,154 @@
+{{! template-lint-disable no-inline-styles }}
+<EmberTable
+    style={{this.height}}
+    @tableId={{this.tableId}}
+    @tableClasses={{this.tableClassNames}}
+    as |T|
+>
+    <T.head
+        @columns={{this.columns}}
+        @enableResize={{this.enableResize}}
+        @enableReorder={{this.enableReorder}}
+        @widthConstraint={{this.widthConstraint}}
+        @fillMode={{this.fillMode}}
+        @fillColumnIndex={{this.adjustedFillColumnIndex}}
+        @resizeMode={{this.resizeMode}}
+        @sortFunction={{this.sortFunction}}
+        @onHeaderAction={{this.onHeaderAction}}
+        @onUpdateSorts={{if this.enableSort this.onUpdateSorts}}
+        @onReorder={{this.onReorder}}
+        @onResize={{this.onResize}}
+        @sorts={{this.sorts}}
+        as |Head|
+    >
+        <Head.row as |Row|>
+            <Row.cell
+                class={{concat
+                    Row.columnValue.headerClassNames
+                    (if Row.columnValue.isFixedLeft " data-table-col-fixed-left")
+                    (if (eq Row.columnValue this.firstVisibleNonFixedColumn) " data-table-col-first-nonfixed")
+                }}
+                as |columnValue columnMeta|
+            >
+                {{#if (and columnValue.headerComponent columnValue.isVisible)}}
```

Comment:
i would probably add comments above these lines where we use the `isVisible` prop, so we know where we are using this temp fix when the bug gets fixed.